x-pack/plugins/reporting/index.js

```diff
@@ -100,6 +100,7 @@ export const reporting = (kibana) => {
               otherwise: Joi.default(false),
             }),
             chromium: Joi.object({
+              inspect: Joi.boolean().default(false),
```

Comment:
Can we make this a valid config only if we're in a dev environment?
```
              inspect: Joi.boolean().when('$dev', {
                autoDownload: Joi.boolean().when('$dev', {
                  is: false,
                  then: Joi.valid(false),
                  else: Joi.default(false)
                }),
              }),
```

I'm not 100% about this, but if we can add safety like this, then we can make its existence well-known and documented. Only if there's not a chance of this being enabled in production though.

Reply:
Nice find, didn't know about this!